Do final edits to proposal (#3)
cct-datascience/volcalc-isc-proposal: proposal/problemdefinition.Rmd

Volatile organic compounds are important components of many biological and chemical processes, yet estimates of compound volatility are not available for most compounds. 

> These compounds are chemicals that readily evaporate under ambient conditions, and are important in a number of fields and contexts including plant defense against herbivores, as routes of microbial communication, and as important indoor pollutants, to name a few.

Reviewer: Add "(VOC)" above as abbreviation for volatile organic compounds.  Then...
```suggestion
VOCs readily evaporate under ambient conditions and are important in a number of fields and contexts including plant defense, microbial communication, and indoor air pollution, to name a few.
```

> Yet measures of volatility are time consuming to calculate experimentally, and volatility estimates are not available for the vast majority of chemical compounds in chemical information databases. 

Reviewer: ```suggestion
Yet measures of volatility are time consuming to calculate experimentally and not available for the vast majority of compounds in chemical information databases. 
```

> The `volcalc` R package automates the process of estimating volatility of compounds. 
> It uses a method for predicting volatility from the chemical structure of the compound [@pankowSIMPOLSimpleGroup2008] to do so.

Reviewer: ```suggestion
The `volcalc` R package automates the process of estimating volatility of compounds from their chemical structure [@pankowSIMPOLSimpleGroup2008].
```

> Our package automates these following steps: 1) downloading data on chemical structure, 2) parsing those data to discover chemical functional groups, 3) applying the SIMPOL algorithm to predict volatility from functional groups and molecular weight, and assign compounds a volatility category given a reference environment. 

Reviewer: ```suggestion
Our package automates these following steps: 1) downloading data on chemical structure, 2) parsing those data to discover chemical functional groups, 3) applying the SIMPOL algorithm to predict volatility from functional groups and molecular weight, and assigning compounds a volatility category given a reference environment. 
```

This enables fast and easy estimation of volatility for thousands of chemical compounds. 

> While `volcalc` does this successfully, there are some limitations to its use that we would like to expand. 
> As the package currently is limited to working with compounds present in a single database and one type of file for describing compound structure, we would like to expand its use to additional compounds in more databases and more file types. 
> The package will also be more usable and reliable for researchers interested in volatility with an expansion of testing and documentation. 

Reviewer: ```suggestion
`volcalc` is currently limited to working with compounds present in a single database. We propose expanding its use to include any chemical with structural information—something widely available in many chemical information databases. In addition, we will improve testing and documentation to make the package more reliable.
```

> This proposal focuses on expanding the scope of `volcalc` and preparing it for a larger and more diverse user base so anyone interested in volatile organic compounds can integrate it into their workflow.

Reviewer: repetitive of previous two sentences now.

Author: Removed in 4b1213c
